### PR TITLE
DAWN 2.16.0 release

### PR DESCRIPTION
**Pull request by PeterC-DLS**
Need to build the mac dmg too

### CODE REVIEW

**Comment by tschoonj** [comment]
I will build the DMG at home... still doesn't work on my iMac 😢

File: _pages/02_download-dawn.md [hunk 1]
```diff
@@ -73,5 +73,5 @@ To find the DAWN version you are currently using, go to Help&rarr;About DAWN Sci
 |               | Windows 7            | Supported (Diamond)                                                      |
 |               | Windows XP           | Not Supported                                                            |
 |               | Other                | Not supported (Windows 98/ME/NT known not to work, Vista normally is ok) |
-| **macOS**     | 10.8 to 10.14        | Beta version available. Would like to officially support eventually.     |
+| **macOS**     | 10.8 to 10.15        | Beta version available. Would like to officially support eventually.     |
```
Comment (tschoonj) [inline _pages/02_download-dawn.md]:
```suggestion
| **macOS**     | 10.8 to 10.14        | Beta version available. Would like to officially support eventually.     |
```

I don't have access to Catalina at the moment to test it.

Comment (tschoonj) [reply _pages/02_download-dawn.md]: @PeterC-DLS  The dmg is in place, so feel free to commit this change in, and I will merge in the PR.

Comment (PeterC-DLS) [reply _pages/02_download-dawn.md]: Ok, I will add an extra line for Catalina saying it is untested.

**Comment by tschoonj** [comment]
Thanks Pete!

**Pull request merged**